src/scenes.jl

```diff
@@ -1,14 +1,19 @@
 
 """
-    Scene TODO document this
+    A Scene can be understood as a (sub) window.
+    It opens up a rectangle in pixels (pixelarea(scene)) on the current screen,
+    and a camera that maps from data space to the pixel in the rectangle.
+    It also holds all graphical objects that are drawn in that area.
+    It currently also holds the display events (mouse position etc),
+    but this may change!
```

Comment:
```suggestion
    Scene
    
A Scene can be understood as a (sub) window.
It opens up a rectangle in pixels (`pixelarea(scene)`) on the current screen,
and a camera that maps from data space to the pixels in the rectangle.
It also holds all graphical objects that are drawn in that area.
Currently, it holds the display events (mouse position, etc.),
but this may change!
```